### PR TITLE
Remove redundant KernelPicker type setting

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1947,30 +1947,6 @@
                     "default": false,
                     "markdownDescription": "%jupyter.configuration.jupyter.enableExtendedKernelCompletions.markdownDescription%",
                     "scope": "machine"
-                },
-                "jupyter.showOnlyOneTypeOfKernel": {
```
Comment (DonJayamanne) [inline package.json:1951]: Old setting was never used, hence no point leaving this around

File: package.json [hunk 2]
```diff
@@ -1947,30 +1947,6 @@
...
-                    "default": false,
-                    "description": "%jupyter.configuration.jupyter.showOnlyOneTypeOfKernel.markdownDescription%",
-                    "deprecationMessage": "%jupyter.configuration.jupyter.showOnlyOneTypeOfKernel.deprecationMessage%",
-                    "scope": "application",
-                    "tags": [
-                        "experimental"
-                    ]
-                },
-                "jupyter.experimental.kernelPickerType": {
-                    "type": "string",
```
Comment (DonJayamanne) [inline package.json:1962]: Removing this

File: src/platform/common/featureManager.ts [hunk 3]
```diff
@@ -84,10 +84,7 @@ export class FeatureManager implements IFeaturesManager {
 
     private _updateFeatures() {
         const kernelPickerType =
-            this.workspace.getConfiguration('notebook.kernelPicker').get('type') === 'mru' ||
-            this.configService.getSettings().kernelPickerType === 'Insiders'
```
Comment (DonJayamanne) [inline src/platform/common/featureManager.ts:88]: @rebornix didn't you want to use a single setting for this?
I don't see why we need both VS Code and Jupyter setting, if we want to use just one setting

Comment (rebornix) [reply src/platform/common/featureManager.ts:88]: @DonJayamanne yes, we should only use one from VS Code.

**Review by rebornix — APPROVED** [review]

**Pull request merged**